src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,111 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Continuously asks the user for input and handles the inputs
+ *
+ * @author Sean Lam
+ */
+public class Ui {
+    TaskList itemList;
+
+    public Ui(TaskList itemList) {
+        this.itemList = itemList;
+    }
+
+    public void getUserInputs() {
+        Scanner sc = new Scanner(System.in);
+        int index;
+        System.out.print("Hello! I'm duke.Duke \nWhat can I do for you? \n");
+
+        String input = sc.nextLine();
+        boolean carryOn = true;
```

Comment:
Do you think this boolean can be named more appropriately here (i.e. `canCarryOn`)?